indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -243,7 +243,7 @@
         :let [key (key humanized-field)
               key-name (-> key
                            name
-                           (s/replace #"(\.humanized2|\.humanized|-sn|-id)" ""))
+                           (s/replace #"(\.humanized(_?2)?|-sn|-id)" ""))
```

Comment:
I'm pretty sure I tried to do something like this originally but couldn't because I am bad with regexes - nice